src/boot/ast.ml

```diff
@@ -109,6 +109,7 @@ and const =
 | Ceqs of int option
 (* External functions TODO: Should not be part of core language *)
 | CExt of tm Extast.ext
+| CPy of Pyast.py
```

Comment:
It might be nice to structure both externals in the same way, more or less to the point that both could have identical `.mli` files. Not sure it's worth it to actually create such files, but making all externally used names the same between them is probably nice for consistency.

Reply:
Fixed.